build/SharedFx.targets

```diff
@@ -10,6 +10,7 @@
 
     <FxProjectToBuild Condition=" '$(BuildSiteExtensions)' == 'true' " Include="$(RepositoryRoot)src\SiteExtensions\Runtime\Microsoft.AspNetCore.Runtime.SiteExtension.pkgproj" />
     <FxProjectToBuild Condition=" '$(BuildSiteExtensions)' == 'true' " Include="$(RepositoryRoot)src\SiteExtensions\LoggingBranch\LoggingBranch.csproj" />
+    <FxProjectToBuild Include="$(RepositoryRoot)src\Framework\src\Microsoft.AspNetCore.App.Runtime.csproj" />
```

Comment:
Is this reverting part of e7b00a5508 ? Specifically, @natemcmaster's removal of
``` xml
<FxProjectToBuild Condition=" '$(BuildRuntimeArchive)' != 'false' " Include="$(RepositoryRoot)src\Installers\Archive\*.*proj" />
```

Either way, I'd appreciate more context on this.

Reply:
Yes, it reverts that part. There is more context in the original issue. 
Microsoft.AspNetCore.App.Runtime.win-x64 is packaged during the main build but we don't run the full build for x86 and that's why 
Microsoft.AspNetCore.App.Runtime.win-x86 is not being produced. I attempted to add 
Microsoft.AspNetCore.App.Runtime back to shared framework project list so it gets built in win-x86 build.